service/controller/v12/key/key.go

```diff
@@ -217,6 +218,14 @@ func DNSZones(customObject providerv1alpha1.AzureConfig) providerv1alpha1.AzureC
 	return customObject.Spec.Azure.DNSZones
 }
 
+func DefaultIngressPIPName(customObject providerv1alpha1.AzureConfig) string {
+	return fmt.Sprintf("%s-Ingress-PublicLoadBalancer-PublicIP", ClusterID(customObject))
```

Comment:
This is the desired name for PIP now.

Reply:
Does it have to prefixed with cluster ID? Isn't it enough to put in the cluster RG?

Reply:
Technically, load balancer name should be unique on a [resource group scope](https://docs.microsoft.com/en-us/azure/architecture/best-practices/resource-naming#networking), but I would also consider that having a bit more explicit name can make our life easier when looking at the list of resources (Azure portal, CLI and similar). We are also prefixing other (resource group scoped) Azure resources with cluster ID, although it's not technically necessary.

Reply:
IMO shorter IDs also make life easier. I personally don't look at the full list of resources. My 2c. Not blocker.

Reply:
> IMO shorter IDs also make life easier.

Good point, couldn't disagree.

In general, prefixed resource names or not, my (non-blocking) vote goes for consistency.

Reply:
I followed the pattern of other two LBs name here: https://github.com/giantswarm/azure-operator/blob/master/service/controller/v12/resource/deployment/template/public_load_balancer_setup.json#L36

Reply:
Don't sped too much time on this 👍